evalml/tests/component_tests/test_pca.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+from evalml.pipelines.components import PCA
+
+
+def test_pca_numeric():
+    X = pd.DataFrame([[3, 0, 1, 6],
+                      [1, 2, 1, 6],
+                      [10, 2, 1, 6],
+                      [10, 2, 2, 5],
+                      [6, 2, 2, 5]])
+    pca = PCA()
+    expected_X_t = pd.DataFrame([[3.176246, 1.282616],
```

Comment:
Just curious, how did you get these expected values? Wondering if there's a way to write / store these values in a way that will make more sense than just a 3d list of floats :O

Reply:
Hm, I'm not sure if there's a better way to write these numbers! PCA centers features and projects them down to a smaller vector space, so there's no simple or easy to anticipate relationship between the input and output features. Very open to any suggestions though!